src/http/app.ts

```diff
@@ -153,15 +155,21 @@ app.get("/data/:chainId/rounds/:roundId/vote_coefficients.csv", async (req, res)
   try {
     const [votes, data] = await Promise.all([
       db.collection(`rounds/${roundId}/votes`).all(),
-      fs.promises.readFile("./data/passport_scores.json", { encoding: "utf8", flag: "r" }),
+      fs.promises.readFile("./data/passport_valid_addresses.json", { encoding: "utf8", flag: "r" }),
     ]);
 
     const passportScores = JSON.parse(data);
     const passportScoresMap = processPassportScores(passportScores);
 
     const records = votes.map((vote: any) => {
       const voter = vote.voter.toLowerCase();
-      const combinedVote = { ...vote, ...passportScoresMap[voter] ?? {} };
+      const score = passportScoresMap[voter];
+      const coefficient = score !== undefined ? 1 : 0;
+      const combinedVote = {
+      ...vote,
+      ...(score ?? {}),
```

Comment:
score here is `bool` or `undefined` so it cannot be spread

Reply:
right I left the same as before, but from the other comment I realized it's better to return all the scores as you said. I'll change it now